baker/SiteBaker.tsx

```diff
@@ -67,24 +66,78 @@ import sharp from "sharp"
 import { generateEmbedSnippet } from "../site/viteUtils.js"
 import { BAKED_BASE_URL } from "../settings/clientSettings.js"
 
+// These aren't all "wordpress" steps
+// But they're only run when you have the full stack available
+const wordpressSteps = [
+    "assets",
+    "blogIndex",
+    "embeds",
+    "googleScholar",
+    "redirects",
+    "rss",
+    "variables",
```

Comment:
We don't use `variables` anywhere. We bake variables as part of baking charts, so I'd leave this one out (we won't be baking variables soon anyway).